erpnext/setup/install.py

```diff
@@ -103,3 +104,45 @@ def add_company_to_session_defaults():
 		"ref_doctype": "Company"
 	})
 	settings.save()
+
+def add_standard_navbar_items():
+	navbar_settings = frappe.get_single("Navbar Settings")
+
+	erpnext_navbar_items = [
+		{
+			'item_label': 'Documentation',
+			'item_type': 'Route',
+			'route': 'https://erpnext.com/docs/user/manual',
+			'is_standard': 1
+		},
+		{
+			'item_label': 'User Forum',
+			'item_type': 'Route',
+			'route': 'https://discuss.erpnext.com',
+			'is_standard': 1
+		},
+		{
+			'item_label': 'Report an Issue',
+			'item_type': 'Route',
+			'route': 'https://github.com/frappe/erpnext/issues',
+			'is_standard': 1
+		}
+	]
+
+	current_nabvar_items = navbar_settings.help_dropdown
+	navbar_settings.set('help_dropdown', [])
+
+	for item in erpnext_navbar_items:
+		navbar_settings.append('help_dropdown', item)
+
+	for item in current_nabvar_items:
+		navbar_settings.append('help_dropdown', {
+			'item_label': item.item_label,
+			'item_type': item.item_type,
+			'route': item.route,
+			'action': item.action,
+			'is_standard': item.is_standard,
+			'hidden': item.hidden
+		})
```

Comment:
```suggestion
	navbar_settings.extend('help_dropdown', erpnext_navbar_items)
	navbar_settings.extend('help_dropdown', current_nabvar_items)
```

Reply:
> 

I tried your suggestion and it seems to mess up the ordering of the items in the dropdown. I am not exactly sure why but my guess is if we simply extend, other doc properties like `idx` etc. are still there which may be the reason of mess up the ordering

Reply:
Weird... I think `extend` is implemented just your for-loop.  🤔

Reply:
Yes it is, but in my for loop above I am not passing any other doc attributes like `idx` etc. But while extending with `current_navbar_items` all those are also passed and it messes up the ordering